src/components/feature-board/Board.unit.ts

```diff
@@ -1,11 +1,10 @@
 import createComponentMocks from "@@/tests/test-utils/componentMocks";
 import { shallowMount, Wrapper } from "@vue/test-utils";
-import { ref } from "vue";
+import Vue, { reactive, ref } from "vue";
```

Comment:
```suggestion
import Vue, { ref } from "vue";
```
reactive is not used